enhancements/update/update-status-api.md

```diff
@@ -0,0 +1,398 @@
+        _: ...
+      - type: UpdateHealth # General update health insight
+        _: ...
+    conditions: <list of standard kubernetes conditions>
+  workerPools:
+  - name: workers
+    _: ...
+    informers: <list of informers with reported insights>
+    conditions: <list of standard kubernetes conditions>
+  - name: infra
+    _: ...
+  conditions: <list of standard kubernetes conditions>    
+```
+
+The API has three conceptual layers:
+
+1. Through the innermost layer `.status.{controlPlane,workerPools[]}.informers`, the API exposes detailed information about individual concerns related to the update, called "Update Insights." The API is prepared to allow multiple external informers to contribute insights, but in this enhancement, the only informer is the USC itself.
+1. The aggregation layer `.status.{controlPlane,workerPools[]}` reports higher-level information about the update through this layer, serving as the USC's interpretation of all insights.
+1. The outermost layer, `.status.conditions`, is used to report operational matters related to the USC itself (the health of the controller and gathering the insights, not the health of the update process).
+
+We do not expect users to interact with `UpdateStatus` resources directly; the API is intended to be used mainly by tooling. A typical `UpdateStatus` instance is likely to be quite verbose.
+
+#### Update Insights
+
+Update Insights are units of update status/health information. There are two types of update insights: status insights and health insights.
+
+##### Update Status Insights
+ 
+Update Status Insights are directly tied to the update process, regardless of whether it is proceeding smoothly or not. Status Insights expose the status of a single resource that is directly involved in the update process, usually a resource that:
+
+1. Either has a notion of "being updated," such as a `Node` or `ClusterOperator`
+1. or represents a higher-level abstraction, such as a `ClusterVersion` resource (represents the control plane) or `MachineConfigPool` (represents a pool of nodes).
+
+Initially, the USC will produce status insights for `ClusterVersion`, `ClusterOperator`, `MachineConfigPool`, and `Node` resources.
+
+##### Update Health Insights
+
+Update Health Insights report a state or condition in the cluster that is abnormal or negative and either affects or is affected by the update. Ideally, none would be generated in a standard healthy update. Health insights should communicate a condition that requires immediate attention by the cluster administrator and should be actionable. Links to resources helpful in the given situation should accompany health insights. Health insights can reference multiple resources involved in the situation. For example, a health insight exposing a failure to drain a node would reference the `Node`, the `Pod` that fails to be drained, and the `PodDisruptionBudget` that protects it. Individual resources can be involved in multiple insights.
```

Comment:
nit: "requires immediate attention" might be too strong.  For example, the impact of a PDB-protected workload blocking a Node drain might just be "Node drain doesn't progress until the PDB issue resolves", which might just be "wait for the PDB-protected CI run to wrap up".  Can we soften to something like "that warrants attention by the cluster administrator" so we have room for things like PDB-vs-drain that sit on the border between "admin might care" and "admin might not care, but informing them is still expected to be acceptable signal-to-noise"?